Hacktoberfest_2022_contributors_guidelines.MD

```diff
@@ -0,0 +1,15 @@
+# Hacktoberfest 2022 contributors guidelines
+
+Just as we’re getting ready for the Hacktoberfest 2022 hype, allow us to remind you of a few guidelines to follow to make sure the event goes smoothly for everyone. These guidelines are there to support the already existing ones available on Hacktoberfest's [](https://hacktoberfest.digitalocean.com/resources)[official website](https://hacktoberfest.com). 
+
+📜 Hacktoberfest is a highly intense time for both organisers and participants. To optimise time and ensure we can get to your PR in the best possible delays, we won’t focus on the PR that don’t meet the quality standards shared on this year’s Hacktober [website](https://hacktoberfest.com/participation/#spam). Indeed, the significant number of entries will force us to prioritise our tasks as we hope to review as much of our contributors’ hard work as possible.
+
+🔍 In addition to that, and to respect our team's work and the time they invest in reviewing your contributions, please make sure to incorporate all the feedback they have already offered before asking for additional feedback. If any of the comments they shared are unclear to you, please feel free to ask for clarification. Still, our team won't review again a PR that didn't take the globality of the initial feedback into account during the revision.
+
+⏰ We couldn’t be more pleased to see a high number of contributions! We kindly request your patience with us; we will focus on reviewing your PRs from Monday to Thursday, during usual working hours, CEST time. If you’re outside that timeframe, please don’t panic; we will get to yours soon.
+
+👑 October is a special time to celebrate the open source community and its contributors; thus, we wish to put coding-oriented entries under the spotlight. Documentation PRs will not be opened this time around, as we feel like Hacktoberfest might not be the most relevant time of the year for them.
+
```

Comment:
I don't think we only want to `put coding-oriented entries under the spotlight.` I'd say something about non-code contributions, that the right place for them would be the `Awesome Meilisearch` repo. But indeed saying that we won't make the `Documentation` repo available is great.

Reply:
Reworded it!

Reply:
there's a typo on `codeing`, but apart from this, lgtm. Congrats @Luna-meili ! You can now submit it to the post process and get a review from the editorial team :)